IBX-517: Updated created content and assign tags (#140)
ezsystems/user-documentation: docs/taxonomy.md

### Assign tag to content from taxonomy tree

Taxonomy is controlled and configured by the Field type **Taxonomy Entry Assignment** settings, so you can use multiple instances

> of the Field type with different taxonomies, for example, tags and product categories in the same Content Type. 

Reviewer: ```suggestion
of the Field with different taxonomies, for example, tags and product categories in the same Content Type. 
```

> Taxonomy is controlled and configured by the Field type **Taxonomy Entry Assignment** settings, so you can use multiple instances
> of the Field type with different taxonomies, for example, tags and product categories in the same Content Type. 

Reviewer: actually, even better:
```suggestion
Taxonomy is controlled and configured by the **Taxonomy Entry Assignment** Field settings,
so you can use multiple Fields of this type with different taxonomies,
for example, tags and product categories in the same Content Type. 
```

> To be able to assign tags to a Content, first, you need to add a Taxonomy Field type.

Reviewer 2: Please refer to the fieldtype as `Taxonomy Entry Assignment Fieldtype` because there is another fieldtype called `Taxonomy Entry FieldType` which may be confusing to some.

> To be able to assign tags to a Content, first, you need to add a Taxonomy Field type.

Reviewer: ```suggestion
To be able to assign tags to a Content, first, you need to add a Taxonomy Field to the Content Type definition.
```
You don't add Field _Types_, but Fields

1. Go to **Content Types** -> **Content**.

> 1. Click the **Edit** icon next to the Contet Type you want to modify.

Reviewer: ```suggestion
1. Click the **Edit** icon next to the Content Type you want to modify.
```

1. Go to **Field definitions** and from the available options, drag and drop the **Taxonomy Entry Assignment** to fields in the Content Type.

> 1. From the **Taxonomy** drop-down, select the taxonomy you want to tag this Content with.

Reviewer: ```suggestion
1. From the **Taxonomy** drop-down, select the taxonomy you want to tag this Content item with.
```

> Now, when you edit or create a Content Type, in the tags section, you can add a tag by clicking **Select Taxonomy Entries**.

Reviewer: ```suggestion
Now, when you edit or create a Content item of this type, in the tags section, you can add a tag by clicking **Select Taxonomy Entries**.
```
